tests/nlu/test_utils.py

```diff
@@ -32,7 +32,10 @@ def fake_model_dir(empty_model_dir):
 
 def test_relative_normpath():
```

Comment:
we could also update the utils function to use `Path(p).resolve()` - it replaces `os.path.normpath()` as of py 3.6: https://docs.python.org/3.6/library/pathlib.html#pathlib.Path.resolve